cmake/DownloadLibSodium.cmake

```diff
@@ -12,12 +12,24 @@ file(MAKE_DIRECTORY ${LIBSODIUM_PREFIX}/include)
 
 include(ExternalProject)
 include(ProcessorCount)
+
+if (CMAKE_C_COMPILER_ARG1)
```

Comment:
What?

Reply:
cmake doesn't pass through `-m[32|64]` to autotools if the user specified it in `CMAKE_C_COMPILER`/`ENV{CC}`